Systems/c172p-glass-effects.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0"?>
+
+<!--
+
+  c172p glass effects.
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program; if not, write to the Free Software
+    Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+
+-->
+
+<system name="c172p-glass-effects">
+
+    <channel name="rain">
+
+        <fcs_function name="glass-effects/splashx">
+            <function>
+                <difference>
+                    <value>-0.1</value>
+                        <product>
+                            <value>2.0</value>
+                            <sqrt>
```

Comment:
It seems the whole `<sqrt>` block is the same in the `splashz` function as well. Is this always true? If yes, then you should move it to a separate `<fcs_function>` above the current two functions.